engine/schema/src/main/java/com/cloud/offerings/NetworkOfferingVO.java

```diff
@@ -485,17 +485,17 @@ public void setIsPersistent(Boolean isPersistent) {
     }
 
     @Override
-    public boolean getIsPersistent() {
+    public boolean isPersistent() {
         return isPersistent;
```

Comment:
This cause the GC bug, our DB framework assumes that the attributes (map) is created by a method interceptor so the name of the variable should not have the 'is', the name of the variable that related to a db table's column should be something after removing get/is string prefix: https://github.com/apache/cloudstack/blob/master/framework/db/src/main/java/com/cloud/utils/db/SearchBase.java#L480
/cc @nvazquez @anuragaw @sh0wstopper @dhlaluku https://github.com/apache/cloudstack/issues/2935